.github/workflows/tests.yaml

```diff
@@ -61,22 +60,62 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
 
+
+  # Define target for integration tests and benchmarks
+  # based on PR title labels
+  target-machine:
+    runs-on: ubuntu-latest
+    outputs:
+      evm: ${{ steps.evm.outputs.machine }}
+      eravm: ${{ steps.eravm.outputs.machine }}
+      default: ${{ steps.default.outputs.machine }}
+    steps:
+
+      - name: Check for EraVM target
+        id: eravm
+        if: contains(github.event.pull_request.title, '[EraVM]')
```

Comment:
@hedgar2017 , could you give me an example?

Reply:
@antonbaliasnikov just like my template above:
`fix: EVM: super mega ultra bug`.

Reply:
`[Target]` -> `Target:` would probably do.

Reply:
So if we ever generate those changelogs, the entry will be `EVM: super mega ultra bug`.

Reply:
@hedgar2017 ,

Ah, got you. Then, I believe `EraVM` and `EVM` should go to [the scope](https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-scope) of the conventional commit.

Here are some examples of commits with scopes:
<img width="781" alt="image" src="https://github.com/user-attachments/assets/a5965d55-0971-434f-9aec-e6adc87cfffe">

In our case it can be:
* `feat(EVM): new feature for evm`
* `fix(EraVM): fix bug in eravm`

So, we can use `(EVM)` and `(EraVM)` strings as conditions. In this case, it will prevent some wrong matches when somebody puts `EraVM` or `EVM` string inside the commit title without wanting to run tests for it.

What do you think?

Reply:
And this scope will be parsed automatically by the semantic release tools and taken into the changelog as you mentioned:
> So if we ever generate those changelogs, the entry will be EVM: super mega ultra bug.